src/FileAdder/FileAdder.php

```diff
@@ -331,4 +324,30 @@ protected function sanitizeFileName(string $fileName) : string
     {
         return str_replace(['#', '/', '\\'], '-', $fileName);
     }
+
+    /**
+     * Attach media to the model.
+     *
+     * @param $media
+     */
+    protected function attachMedia($media)
+    {
+        if (! $this->subject->exists) {
+            $this->subject->unsavedMedias[] = $media;
+
+            $class = get_class($this->subject);
+
+            $class::created(function ($model) {
+                if (! empty($model->unsavedMedias)) {
```

Comment:
Refactor this to bail out with a `return` statement if `model->unsavedMedias` is empty. The the actual process loses an indentation level.